Makefile

```diff
@@ -88,17 +88,17 @@ include $(SUPPORTDIR)/build.mk
 $(SUPPORTDIR)/build.mk: W3CTRMANIFEST $(SUPPORTDIR)
 	@printf ' $(foreach f,$(BUILD_INPUT),$(BUILDDIR)/$(f): $(f) $(BUILDDIR)\n\t@mkdir -p $$(dir $$@)\n\tcp -f $$< $$@\n\n)' > $@
 
-# respec2html needs an X server running
 $(OUTPUT): $(INPUT) $(RESPEC_INSTALL) $(BUILD_FILES) $(BUILDDIR)
-	node $(SUPPORTDIR)/respec/tools/respec2html.js -e --src file://`pwd`/$< --out $@
+	python2.7 -m SimpleHTTPServer 8000 &
```

Comment:
This is going to crash on a number of machines when run locally, since 8000 is a popular number for varioius other setups. You should also bind the server to localhost:8000 rather than just 8000 (if SimpleHTTPServer allows that), because otherwise you're serving this directory to anyone who can reach your machine.

Reply:
@alvestrand suggestion implemented using Python 3 http.server module which allows to bind to a given address (localhost here).
I also used port 8765 instead of 8000
Thanks for the feedback !